src/Expander/Monad.hs

```diff
@@ -860,10 +861,10 @@ setTasks = modifyState . set expanderTasks
 clearTasks :: Expand ()
 clearTasks = modifyState $ set expanderTasks []
 
-expandEval :: Eval a -> Expand a
+expandEval :: Core -> Expand Value
 expandEval evalAction = do
   env <- currentEnv
-  out <- liftIO $ runExceptT $ runReaderT (runEval evalAction) env
+  let out = Right $ evaluateIn env evalAction
   case out of
     Left err -> do
```

Comment:
@coderabbitai in Haskell, it is really strange to use an explicit constructor (in this case `Right`) and then to immediately pattern-match to see if the constructor is `Right` or something else. Please flag this kind of code in the future.

Reply:
Please remember this using the Learning feature.

Reply:
@coderabbitai please remember this using the Learning feature.